terraform/modules/aws/node_group/main.tf

```
@@ -265,7 +265,9 @@ resource "aws_launch_configuration" "node_launch_configuration" {
   instance_type = "${var.instance_type}"
   user_data     = "${join("\n\n", list(file("${path.module}/${var.instance_user_data}"), var.instance_additional_user_data))}"
 
-  security_groups = ["${var.instance_security_group_ids}"]
+  # this awkward syntax should work in both v0.11 and v0.12
+  # (see https://stackoverflow.com/questions/57117183/terraform-0-11-list-attribute-compatible-with-terraform-0-12)
+  security_groups = "${flatten(var.instance_security_group_ids)}"
```

Can these be changed to TF 0.12+ syntax or is there something preventing that?

e.g.
```suggestion
  security_groups = flatten(var.instance_security_group_ids)
```

---

sorry ignore me, makes sense now that I've read the description properly!

---

See the [previous comment](https://github.com/alphagov/govuk-aws/pull/1705#discussion_r1101243875) - there will be other projects using `node_group` which may still be using 0.11.

---

Makes total sense, sorry for the noise! I'll go have another coffee 🙈